OpenRA.Mods.Common/Traits/Render/WithAttackAnimation.cs

```diff
@@ -53,12 +54,20 @@ public WithAttackAnimation(ActorInitializer init, WithAttackAnimationInfo info)
 			armament = init.Self.TraitsImplementing<Armament>()
 				.Single(a => a.Info.Name == info.Armament);
 			wsb = init.Self.Trait<WithSpriteBody>();
+
+			if (init.Self.TraitOrDefault<AttackTurreted>() != null)
```

Comment:
Ah right, that makes sense. I wonder if we want to ensure in some way that we use the correct turret though (e.g. by checking for the `Turret` string, see https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Traits/Turreted.cs#L21 and https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Traits/Render/WithSpriteTurret.cs#L32).